src/NzbDrone.Core/Indexers/Definitions/Cardigann/CardigannRequestGenerator.cs

```diff
@@ -1013,7 +1013,7 @@ private IEnumerable<IndexerRequest> GetRequest(Dictionary<string, object> variab
 
                 // build search URL
                 // HttpUtility.UrlPathEncode seems to only encode spaces, we use UrlEncode and replace + with %20 as a workaround
-                var searchUrl = ResolvePath(ApplyGoTemplateText(searchPath.Path, variables, WebUtility.UrlEncode).Replace("+", "%20")).AbsoluteUri;
+                var searchUrl = ResolvePath(ApplyGoTemplateText(searchPath.Path, variables, Uri.EscapeDataString).Replace("+", "%20")).AbsoluteUri;
```

Comment:
`EscapeDataString` replaces + with %20 already right? So we can drop the extra replace and the note above

Reply:
Off course. Changed